MODULE_EVALUATION_TEMPLATE.MD

```diff
@@ -44,16 +51,17 @@ When performing a technical evaluation of a module, create a copy of this docume
 * [ ] Module includes executable implementations of all endpoints in the provides section of the Module Descriptor
 * [ ] Environment vars are documented in the ModuleDescriptor
   * -_note: read more at [https://wiki.folio.org/pages/viewpage.action?pageId=65110683](https://wiki.folio.org/pages/viewpage.action?pageId=65110683)_
-* [ ] If a module provides interfaces intended to be consumed by other FOLIO Modules, they must be defined in the Module Descriptor "provides" section
-* [ ] All API endpoints are documented in RAML or OpenAPI
+* [ ] If a module provides interfaces intended to be consumed by other FOLIO Modules, they must be defined in the Module Descriptor "provides" section, and must conform to FOLIO [interface naming conventions](https://dev.folio.org/guidelines/naming-conventions/#interfaces).
```

Comment:
That page contains pretty old information (that likely predates governance processes for deciding on the guidance)

At the very least, it provides at least one piece of highly likely to be incorrect guidance:
> [JSON API](https://jsonapi.org/) is de facto standard for defining REST JSON interfaces.

AFAIK, FOLIO _does not_ use JSON API for most of it's HTTP interface definitions. IIRC the only module to widely adopt this approach was [mod-kn-ebsco-ruby](https://github.com/folio-org/deprecated-mod-kb-ebsco-ruby) and that was later rewritten as [mod-kb-ebsco-java](https://github.com/folio-org/mod-kb-ebsco-java) that I think has moved away from this API design

Reply:
Thanks.  @mweaver-ebsco suggested linking to this document.  We can see if TC agrees.  Maybe the best option will be to do so, and then to have some TC (or external) group update that document to address your concern and any other staleness.

Reply:
I've added a [follow-up subgroup item](https://wiki.folio.org/display/TC/TCR+Process+Improvements+-+Fall+2023) to ask TC to address that wiki page.